docs/modding/tools.md

```diff
@@ -0,0 +1,70 @@
+# Modding Tools
+
+## Source Engine
+
+### Titanfall
+
+* [Titanfall VPK Tool](http://cra0kalo.com/public/Titanfall\_VPKTool3.4\_Portable.zip)
+* [Legion](https://wiki.modme.co/wiki/apps/Legion.html)
+
+### RPSNVPK's
+
+* [squidgyberries RSPNVPK](https://github.com/squidgyberries/RSPNVPK)
+* [taskinoz RSPNVPK](https://github.com/taskinoz/RSPNVPK)
+* [Provoxin RSPNVPK](https://github.com/Provoxin/RSPNVPK-GUI)
+
+{% embed url="https://noskill.gitbook.io/titanfall2/how-to-start-modding/modding-introduction/modding-tools/how-to-use-rspnvpk" %}
+NoSkill: How to use RSPNVPKs
+{% endembed %}
+
+### VTF & VMT
+
+* [VTFEdit](this-content-is-no-longer-available.md)
+  * [VTFVer](http://cra0kalo.com/public/VTFVer.zip) _not needed anymore, VTFEdit support v7.5_
+* [VTF Shell Extensions](https://www.wunderboy.org/valve-hl2source-sdk-tools/#vtf\_shell) - [Info](https://developer.valvesoftware.com/wiki/VTF\_Shell\_Extensions)
+
+## Other utilities
+
+### File editor
+
+* [Atom Text](https://atom.io)
+  * [Syntax color KeyValue](https://atom.io/packages/language-source-cfg)
+  * [Syntax color Squirrel](https://atom.io/packages/squirrel-language)
+  * [Color picker](https://atom.io/packages/color-picker)
+* [TextCrawler 3](https://www.digitalvolcano.co.uk/tcdownloads.html)
+* [Visual Studio Code](https://code.visualstudio.com)
+* [Notepad++](https://notepad-plus-plus.org/downloads/)
+
+### Graphics / animation / color editors
+
+* [Paint.net](https://www.getpaint.net)
+  * [Gradient Mapping plug-in](https://forums.getpaint.net/topic/6265-gradient-mapping/)
+  * [Animation Helper](https://pixelbyte.itch.io/paint-net-sprite-plugin)
+* [Source VMT Color](https://dev.cra0kalo.com/?p=155)
+* [GIMP](https://www.gimp.org/downloads/)
+  * [VTF plug-in](https://www.tophattwaffle.com/downloads/gimp-vtf-plugin/)
+* [RGB Tool converter](http://hewmc.blogspot.fr/2012/12/rgb-to-percentage-converter.html)
+* [Color Pick Windows](https://pixelbyte.itch.io/color-pick)
+* [GIF splitter](https://ezgif.com/split)
+* [Rad Tools Bik](http://www.radgametools.com/bnkdown.htm) (needs [Quicktime 7.6](https://support.apple.com/downloads/quicktime))
+
+### Archives
+
+* [7zip](https://www.7-zip.org)
+* [WinRAR](https://www.win-rar.com/download.html?L=10)
```

Comment:
Side note, is there any reason to still use WinRAR? Considering that 7zip can even open `.rar` archives nowadays? ^^